### PR TITLE
Tool to post missing ZTF cutouts to SkyPortal

### PR DESCRIPTION
**Pull request by dmitryduev**
This PR implements a simple script that folks can use to post ZTF cutouts to SkyPortal/Fritz in case those are missing.
[Context: some people sometimes post new ZTF candidates to F without posting the thumbnails. There's currently no fast way for K to detect whether thumbnails are missing on the SP side to post them, that's why we're not doing it in our little broker.]

To be honest, I don't really know where such a tool should belong, perhaps even in Fritz's docs or something. Suggestions welcome.

### CODE REVIEW

File: tools/update_ztf_thumbnails_fritz.py [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+    if isinstance(start_date, str) or isinstance(start_date, int):
+        start_date = datetime.datetime.strptime(str(start_date), "%Y%m%d")
+    if isinstance(end_date, str) or isinstance(end_date, int):
+        end_date = datetime.datetime.strptime(str(end_date), "%Y%m%d")
+
+    if start_date is None:
+        start_date = datetime.datetime.utcnow()
+    if end_date is None:
+        end_date = datetime.datetime.utcnow() + datetime.timedelta(days=1)
+    print(start_date, end_date)
```
Comment (kmshin1397) [inline tools/update_ztf_thumbnails_fritz.py:156]:
```suggestion
```

File: tools/update_ztf_thumbnails_fritz.py [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+
+    if end_date < start_date:
+        raise ValueError("End date must be before start date.")
+
+    # print(get_alert_by_object_id(oid="ZTF21abiuvdk"))
```
Comment (kmshin1397) [inline tools/update_ztf_thumbnails_fritz.py:161]:
```suggestion
```

File: tools/update_ztf_thumbnails_fritz.py [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+    alert = k.query(query=q).get("data")[0]
+
+    return alert
+
+
+def main(
+    start_date: Optional[Union[datetime.datetime, str, int]] = None,
+    end_date: Optional[Union[datetime.datetime, str, int]] = None,
+    num_per_page: int = 100,
+):
```
Comment (kmshin1397) [inline tools/update_ztf_thumbnails_fritz.py:146]: I'd like to see informative docstrings for this function :)

**Review by kmshin1397 — COMMENTED** [review]
Mostly looks good @dmitryduev! I agree that it's not very clear where this live, but the tools folder here seems as good a place as any so I'm fine leaving it here. Could also make a note of it in the Fritz docs I suppose, to let people know it's available but I think the code itself can easily live in Kowalski tools.

File: tools/update_ztf_thumbnails_fritz.py (hunk 3 above)
Comment (dmitryduev) [reply tools/update_ztf_thumbnails_fritz.py:146]: Oops, sorry. Done!

**Review by kmshin1397 — APPROVED** [review]

**Pull request merged**